velox/serializers/PrestoSerializer.cpp

```diff
@@ -4263,6 +4263,34 @@ void PrestoVectorSerde::deserializeSingleColumn(
   *result = row->childAt(0);
 }
 
+void PrestoVectorSerde::serializeSingleColumn(
+    const VectorPtr& vector,
+    const Options* opts,
+    memory::MemoryPool* pool,
+    std::ostream* output) {
+  const auto prestoOptions = toPrestoOptions(opts);
+  VELOX_USER_CHECK_EQ(
+      prestoOptions.compressionKind,
+      common::CompressionKind::CompressionKind_NONE);
+  VELOX_USER_CHECK_EQ(prestoOptions.nullsFirst, false);
+
+  const IndexRange range{0, vector->size()};
```

Comment:
Just confirming that vector->size() can be >1 ? If there is any restriction for vector length then it should be validated.

Reply:
Yes, there is no restriction on vector->size() and it can be greater than 1. This check was mistakenly added before, based on the constant folding Prestissimo PR usecase, and it is not needed in velox.